content/about.md

```diff
@@ -1,14 +1,35 @@
 ---
 title: "About"
-date: 2022-04-24T08:1:00+01:00
 
 tags: []
-author: "Monal"
 ---
 
+Monal originates in 2002 as fork of the SwornIM app. Until 2019 it has been developed by Anu Pokharel. Since then Thilo Molitor took over the development and joined in 2020 with Friedrich Altheide. From initial rewrite of code in the backend the entire app has been upgraded with a modern XMPP engine, new features and future-proof setup. Monal challenges to be the go-to XMPP chat-app for the iOS and macOS platform.
+
+## Monal Team
+
+**Thilo Moltor** - Since 2015 he is contributing to Monal IM, when he also started to help out with the Gajim XMPP Client written in Python. Shortly after this, in early 2016 I started to take over maintenance for several Prosody modules as well as developing newones (i.e. mod_smacks, mod_cloud_notify, mod_csi_battery_saver). Later on, in 2017 Thilo started to implement basic push support in Monal IM along with the required push-app-server. Significantly core contribution to Monal continued since early 2020. Namely:
+- Re-implementation of connection management and integration of new XML parser
+- Modern push notifications
+- New XMPP parser completely eliminating some classes of (security relevant) bugs
+- A generalized PEP ([XEP-0163](https://xmpp.org/extensions/xep-0163.html)) framework which is needed as foundation for the mentioned Bookmark Storage ([XEP-0048](https://xmpp.org/extensions/xep-0048.html)) and PEP Native Bookmarks ([XEP-0402](https://xmpp.org/extensions/xep-0402.html))
+- See commitment after 2021 in the [release notes](https://github.com/monal-im/Monal/releases).
+
+**Friedrich Altheide** - Since 2020 Friedrich stated with a first contribution to improve the geo-location functionality in MonalIM. The interest in the project continued and support towards a re-design of the client, especially when it comes to security and privacy aspects, have been co-developed with Thilo. Furthermore, the entire encryption (OMEMO) implementation has been upgraded.
+
+And of course many people from the XMPP Community! Many thanks for your support in the recent years!
+If you read this and want to support go to this page!
+
+## Publications
```

Comment:
Good idea. Could you please add links to our last two presentations in Berlin?

Reply:
https://github.com/monal-im/monal-im.org/pull/6/commits/416734a8c4389bfa273f5809649210c28f0bde15